app/controllers/about_controller.rb

```diff
@@ -3,11 +3,9 @@ def index
   end
 
   def email
-    # TODO remove once we add captcha to form
-    if helpers.is_user_a_guest?
-      respond_to do |format|
-        format.html { redirect_to root_path }
-      end
+    if params["phone"].to_s.length > 0
```

Comment:
I think we should change the param name to "phony" if that's what we're using for the column in the DB. It may be a little confusing down the line.

Reply:
I changed the db column to phone to be consistent.  Get latest.. but you probably will need to do a 'rails db:rollback' then rerun the migration.